<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordSession</class>
 <widget class="QWidget" name="RecordSession">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="recordButton">
     <property name="text">
      <string>Enregistrer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="saveButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Sauvegarder</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="stopButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>ArrÃªter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="time">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>0:00</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>recordButton</sender>
   <signal>clicked()</signal>
   <receiver>RecordSession</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopButton</sender>
   <signal>clicked()</signal>
   <receiver>RecordSession</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>RecordSession</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start()</slot>
  <slot>stop()</slot>
  <slot>ping()</slot>
  <slot>save()</slot>
 </slots>
</ui>
